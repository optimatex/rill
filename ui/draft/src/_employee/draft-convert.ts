export { convertToHTML, convertFromHTML } from 'draft-convert';
