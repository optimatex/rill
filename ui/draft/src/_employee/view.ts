export {
  React,
  useState,
  useCallback,
  css,
  styled,
  createGlobalStyle,
} from '@rill/view';
