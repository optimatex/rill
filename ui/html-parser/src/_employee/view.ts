export { React, RC } from '@rill/view';
