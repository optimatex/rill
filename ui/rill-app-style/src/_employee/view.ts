export { createGlobalStyle, css } from '@rill/view';
