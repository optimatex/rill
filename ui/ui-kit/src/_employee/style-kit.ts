export {
  // appLayout
  appLayoutCss,
  screenCss,
  sidebarColumnCss,
  flatColumnCss,
  mainColumnCss,
  // layout
  listCss,
  // Brand
  logCss,
  dualLogoCss,
  // Controller
  trivialAppHeaderCss,
  // _menu
  menuWrapCss,
  menuItemCss,
  // Image
  imageCss,
  imagesListCss,
  imagesGridCss,
  imagesLayoutItemCss,
  imageCropperCss,
  // Tags
  tag,
  tagsList,
  // Ui-primal
  // _grid
  gridRowContainerCss,
  gridrillRowContainerCss,
  gridColumnCss,
  // _card
  cardContainerCss,
  postContainerCss,
  postTitleCss,
  panelCss,
  // _buttons
  buttonCss,
  buttonActiveCss,
  buticonCss,
  // _form
  formContainerCss,
  formFieldFlatCss,
  formFieldColumnCss,
  formFieldErrorCss,
  formLabelTextCss,
  inputCss,
  toggleContainerCss,
  toggleContainerActiveCss,
  toggleRoundCss,
  toggleRoundActiveCss,
  // _article
  paragraphCss,
  nameCss,
  richContentWrapCss,
  // _table
  tableContainerCss,
  tableHeadCss,
  tableRowCss,
  tableColumnCss,
  // Unknown
  // _object
  stringValueCss,
  numberValueCss,
  booleanValueCss,
  emptyValueCss,
  arrayWrapCss,
  arrayItemCss,
  objectKeyCss,
  objectWrapCss,
  objectPairCss,
  // Unsorted
  lightIndicatorCss,
  popoverCss,
  pipeCss,
  hintableLogCss,
  fixedPanelCss,
} from '@rill/style-kit';
