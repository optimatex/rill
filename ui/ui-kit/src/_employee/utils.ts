export {
  toPairs,
  find,
  has,
  path,
  prop,
  curry,
  assoc,
  lensProp,
  view,
  set,
} from '@rill/utils';
