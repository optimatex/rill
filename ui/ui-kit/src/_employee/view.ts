export {
  React,
  useState,
  useCallback,
  // styled-components
  styled,
  // Lens
  // getTestId,
  // Types
  FC,
  RC,
  ReactNode,
  LegacyRef,
  // PropTypes
  IContainerProps,
  // Event types
  FormEvent,
  MouseEvent,
} from '@rill/view';
