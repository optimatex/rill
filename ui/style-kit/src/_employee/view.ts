import { css as originalCss } from '@rill/view';

export const css = originalCss;
