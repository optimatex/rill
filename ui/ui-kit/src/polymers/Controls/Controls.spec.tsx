import { React } from '_employee/view';
import { render, mockMenuScheme_1 } from 'test';
import { Controls } from '.';

describe('Controls', () => {
  it('should render without errors', () => {
    const { getAllByRole } = render(<Controls scheme={mockMenuScheme_1} />);
    const el = getAllByRole('button');

    expect(el).toMatchSnapshot();
  });
});
