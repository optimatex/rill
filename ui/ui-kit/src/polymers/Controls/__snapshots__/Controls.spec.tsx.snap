// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Controls should render without errors 1`] = `
Array [
  .c1::before {
  font-size: 22px;
  position: relative;
  color: #6b62b1;
  top: px;
  bottom: 1.5px;
  left: px;
  right: 1.5px;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  box-sizing: border-box;
  padding: 4px 8px;
  border-width: 1px;
  border-style: solid;
  background: red;
  color: white;
  padding: 0;
  border-radius: 50px;
  width: 34px;
  height: 34px;
}

<button
    class="c0"
  >
    <i
      class="c1 icon-rewind"
      color="#6b62b1"
      name="rewind"
      size="22"
    />
  </button>,
  .c1::before {
  font-size: 15px;
  position: relative;
  color: green;
  top: px;
  bottom: px;
  left: px;
  right: px;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  box-sizing: border-box;
  padding: 4px 8px;
  border-width: 1px;
  border-style: solid;
  background: red;
  color: white;
  padding: 0;
  border-radius: 50px;
  width: 34px;
  height: 34px;
}

<button
    class="c0"
  >
    <i
      class="c1 icon-menu"
      color="green"
      name="menu"
      size="15"
    />
  </button>,
  .c1::before {
  font-size: 43px;
  position: relative;
  color: pink;
  top: px;
  bottom: px;
  left: px;
  right: px;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  box-sizing: border-box;
  padding: 4px 8px;
  border-width: 1px;
  border-style: solid;
  background: red;
  color: white;
  padding: 0;
  border-radius: 50px;
  width: 34px;
  height: 34px;
}

<button
    class="c0"
  >
    <i
      class="c1 icon-play"
      color="pink"
      name="play"
      size="43"
    />
  </button>,
]
`;
